src/funding/applepay/config.jsx

```diff
@@ -38,6 +38,8 @@ export function getApplePayConfig() : FundingSourceConfig {
         logoColors:  {
             [ BUTTON_COLOR.BLACK ]:  LOGO_COLOR.WHITE,
             [ BUTTON_COLOR.WHITE ]:  LOGO_COLOR.BLACK
-        }
+        },
+
+        shippingChange: true
```

Comment:
That's cool that ApplePay supports the shippingCallback! 💯

Reply:
We patch in off of Apple's Event listeners.  Now just have to get it working by upgrading LSAT.